src/main/resources/stylesheets/snyk_code_suggestion.scss

```diff
@@ -0,0 +1,80 @@
+::-webkit-scrollbar-thumb {
+  background: var(--scrollbar-thumb-color);
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: #595a5c;
+}
+
+body {
+  color: var(--text-color);
+  font-family: "SF Pro Text", "Segoe UI", "Ubuntu", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+a,
+.link {
+  color: var(--link-color);
+}
+
+.delimiter {
+  border: 1px solid #505254;
+}
+
+
+.ignore-warning {
+  background: #FFF4ED;
+  color: #B6540B;
+  border: 1px solid #E27122;
+}
+
+
+.ignore-badge {
+  background: #FFF4ED;
+  color: #B6540B;
+  border: 1px solid #E27122;
+}
+
+
+.data-flow-body {
+  background-color: var(--data-flow-body-color);
+  font-family: 'Courier New', Courier, monospace;
+}
+
+
+.data-flow-clickable-row {
+  color: var(--link-color);
+}
+
+
+.data-flow-delimiter {
+  color: #BBBBBB;
+}
+
+.tabs-nav {
+  border-bottom: 1px solid var(--tabs-bottom-color);
+}
+
+.tab-item-icon path {
+  fill: var(--tab-item-github-icon-color);
+}
+
+.tab-item:hover {
+  background-color: var(--tab-item-hover-color);
+}
+
+.tab-item.is-selected {
+  border-bottom: 3px solid #3474f0;
+}
+
+.example-line-number,
+.example-line>code {
+  font-family: "Courier New", Courier, monospace;
+}
+
+.example-line.added {
+  background-color: var(--example-line-added-color);
+}
+
+.example-line.removed {
+  background-color: var(--example-line-removed-color);
+}
```

Comment:
Why did we need to use a SASS file? I thought SASS is needed when we need things like nesting or mixins or other SASS [magic](https://sass-lang.com/guide/) ✨

Reply:
Mostly because we use SASS in VSCode so I wanted to be as consistent as possible across the two, but also because there are more features available in SASS (like you said) and in the future we will have more than Snyk Code styling but still want to share stylesheets or colours across Snyk OSS, Snyk IaC, and Snyk Container